### DISCUSSION TITLE
Hide the split line

### DISCUSSION THREAD

**Discussion by pjfichet** [scandum/tintin#224] [Script help]
Hello alls,

I'm using `#split 0 2`, and `#showme` to print stuff on lines -2 and -3. So far, so good. But unfortunately, on startup, and each time my window is redrawn, tintin draws the so called `split line`, and creates a visual mess.

Is it possible to prevent the bottom status bar to be filled by dashes ?

**Comment by scandum** [comment]
Hi there. You should be able to use:
```
#event {CATCH SCREEN SPLIT FILL} {#nop}
```

**Comment by pjfichet** [comment]
Hi !

That's it, thank you !
